audit-service/src/test/java/uk/gov/gchq/palisade/audit/service/request/ReadRequestCompleteAuditRequestTest.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.gchq.palisade.audit.service.request;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.gchq.palisade.Context;
+import uk.gov.gchq.palisade.RequestId;
+import uk.gov.gchq.palisade.User;
+import uk.gov.gchq.palisade.resource.impl.DirectoryResource;
+import uk.gov.gchq.palisade.resource.impl.FileResource;
+import uk.gov.gchq.palisade.resource.impl.SystemResource;
+import uk.gov.gchq.palisade.rule.Rules;
+
+import java.io.IOException;
+import java.util.AbstractMap;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+import static java.util.stream.Collectors.toMap;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+@RunWith(SpringRunner.class)
```

Comment:
Does this need Spring? It looks like it should be using a standard JUnit runner.

Reply:
Changed to JUnit4.class